WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -40,7 +40,8 @@ class CommentContentTableViewCell: UITableViewCell, NibReusable {
     @IBOutlet private weak var replyButton: UIButton!
     @IBOutlet private weak var likeButton: UIButton!
 
-    @IBOutlet private weak var moderationBar: CommentModerationBar!
+    // This is public so its delegate can be set directly.
+    @IBOutlet weak var moderationBar: CommentModerationBar!
```

Comment:
Should we make this `readonly` for public access, since the caller only needs to assign the `delegate` property?

```suggestion
    @IBOutlet private(set) weak var moderationBar: CommentModerationBar!
```